docs/examples/single_moving_target_relative_Doppler.ipynb

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    from numpy.fft import fftn,fftshift
</pre><p>Please remove. I usually don't do this and use <code>np.fft.fft</code> in the code to highlight which FFT-implementation is used. Furthermore, it won't be required anymore as soon as Doppler processing is implemented in <code>radar_scene.py</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rf-eng/scikit-radar/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>